packages/material-ui/src/Select/Select.test.js

```diff
@@ -197,7 +197,7 @@ describe('<Select />', () => {
     const { getByRole } = render(<Select value="" autoFocus />);
 
     act(() => {
```

Comment:
We can remove this now. `fireEvent` uses `act` internally. There are more instances of this in this PR. I think I forgot to remove them. It doesn't have any negative impact but the less code we have the better.